src/app/inventory/store/sockets.ts

```diff
@@ -142,7 +143,10 @@ function buildInstancedSockets(
 
   return {
     allSockets: createdSockets, // Flat list of sockets
-    categories: categories.sort(compareBy((c) => c.category?.index)), // Sockets organized by category
+    categories:
+      itemDef.inventory?.bucketTypeHash === BucketHashes.Subclass
+        ? categories.sort(compareBy((c) => c.category?.index))
+        : categories, // Sockets organized by category
```

Comment:
Not a huge win, but subclasses are the only place where socket category ordering seems to matter in any way someone might care about.